ui_framework/components/global_styles/variables/_z_index.scss

```diff
@@ -0,0 +1,5 @@
+$kuiZContent:         1;
+$kuiZContentMenu:     100;
+$kuiZNavigation:      200;
+$kuiZMask:            300;
+$kuiZModal:           400;
```

Comment:
If i remember correctly, some old browsers cap at 999. If we can't work within a range of 1000 we're in trouble? haha.